### ISSUE TITLE
Error List (Visual Studio Window not Updated)

### ISSUE THREAD

**Issue by toeb** [StefanKert/BuildVision#57]
I have noticed that since i installed BuildVision in VS2017 Pro that the normal Error List window in visual studio does not have its errors updated correctly.  After a build BuildVision shows the errors but the Error List does not show them.  if i build again the errors of the previous build are shown in the error list.  Has anyone else seen this issue as well or is this due to my VS configuration

**Comment by muellerto** [comment]
I don't think this depends on BuildVision. I have this, too, even when BuildVision is not open. It depends somehow on the ErrorList window himself. You must open it once before your first build starts, then it gets the errors. Until you didn't open it it does never get any messages.

**Comment by StefanKert** [comment]
Sorry for the late reply and thanks @muellerto so much for jumping in :)

I think the main difference is the following:

BuildVision adds all the errors that are thrown during a build and as a result, passed through the BuildLogger.  So BuildVision only gets the errors for each Build.

The error list, at least from what I observed, gives you errors without compilation so IMO Error List after you finished one succeeded Build ( so bin and obj are created) gives you the errors IntelliSense shows. 

Long story short: I think the main difference here is that Error List depends on IntelliSense errors and BuildVision depends on Build errors.

(Please correct me if these assumptions are wrong!) 

Anyways, some errors are different and sometimes some errors only occur in Error List so I will do some research if accessing IntelliSense errors from EnvDTE is possible.

**Comment by muellerto** [comment]
The Error List _can_ show IntelliSense stuff (and more). But I have this normally turned off.

I use the Error List (not very often) to filter a long list of build messages on a single project in the solution. But sometimes it's indeed empty, however, BuildVision and the Output window say there are indeed build errors.

This Error List has never been very reliable, the only window you can really trust ist the Output window.
